src/cryptoadvance/specter/templates/wallet_settings.html

```diff
@@ -1,6 +1,13 @@
 {% extends "base.html" %}
 {% block main %}
-<h1>{{wallet.name}}</h1>
+
+<form action="./" method="POST">
+	<h1 id="nametitle">{{wallet.name}}</h1>
+	<input type="text" class="label" autocomplete="off" spellcheck="false" id="walletname" name="walletname" value="{{wallet.name}}" style="font-size: 1.5em; text-align: center; display: none;" />
+	<button type="submit" class="btn update" id="savename" name="action" value="renamewallet" style="display: none; width: 49.2%;">Update</button>
+	<button type="button" class="btn cancel" id="cancelname" onclick="toggleEdit()" style="display: none; width: 49.2%;">Cancel</button>
+	<button type="button" class="btn edit" id="editname" onclick="toggleEdit()">Edit Wallet Name</button>
```

Comment:
Please put buttons in the `<div class="row">` element, otherwise looks bad in Chrome:
<img width="650" alt="Screenshot 2020-04-22 at 02 22 55" src="https://user-images.githubusercontent.com/1706012/79927591-8a393600-8440-11ea-9b4f-6db2bcb39ca0.png">